sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -913,6 +931,86 @@ public Builder get() {
           .setKey(encodedKeyOut.toByteString())
           .setWindow(encodedWindowOut.toByteString()).buildPartial();
     }
-    return cachedPartialBagUserStateKey.toBuilder().setUserStateId(id).build();
+    return StateKey.newBuilder()
+        .setBagUserState(cachedPartialBagUserStateKey.toBuilder().setUserStateId(id).build())
+        .build();
+  }
+
+  private <T, K, V> T bindSideInputView(PCollectionView<T> view) {
```

Comment:
<!--new_thread; commit:d1c0ffde59d0b17126deecb840cbeec36719d7a6; resolved:0-->
K and V are unused

Reply:
Removed K and V.